editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1661,7 +1661,7 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 
 				if (_edit.mode == TRANSFORM_NONE && b->is_pressed()) {
 					if (b->is_alt_pressed()) {
-						if (nav_scheme == NAVIGATION_MAYA) {
+						if (nav_scheme == NAVIGATION_MAYA || spatial_editor->get_tool_mode() != Node3DEditor::TOOL_MODE_SELECT) {
```

Comment:
Should probably be part of the check above instead of here IMO

Reply:
The break needs to occur alt pressed when MAYA navigation is enabled. The break also occurring when the tool mode is not select is inconsequential.

Reply:
Why enter the block at all? Why not replace the `_edit.mode`?

Reply:
I'm not sure I follow. Replace it with what?

Reply:
With the thing you're adding:
```cpp
if (spatial_editor->get_tool_mode() == Node3DEditor::TOOL_MODE_SELECT && b->is_pressed()) {
					if (b->is_alt_pressed()) {
						if (nav_scheme == NAVIGATION_MAYA) {
```

Reply:
That would result in right clicking without alt opening a list select, which would be wrong.

Reply:
How would it do that? The alt check is still there?

Reply:
I seriously need to wear glasses.

Reply:
Ok yeah, but that ends up breaking the ALT + RMB list select when MAYA navigation is enabled.

Reply:
Which is actually a bug outside this PR apparently.

Reply:
Breaking how? It's meant to not work in that mode, that's why the break is there

Reply:
Yeah, you're right I edited my comments.